core/map2.f

```diff
@@ -397,15 +397,24 @@ subroutine read_con(wk,nwk,nelr,nv,ierr)
             call byte_read(hdr,sizeof(hdr)/4,ierr)
             if(ierr.ne.0) goto 100
 
-            read (hdr,*) version,nelgti,nelgvi,nvi
-c    1       format(a5,2i12,i2)
+            read (hdr,'(a5)') version 
+
+            if (version.eq.'#v002') then
+               read (hdr,*) version,nelgti,nelgvi,nvi
+            else
+               read (hdr,1) version,nelgti,nelgvi,nvi
+            endif
+            write (6,*) ' hdr calling ',version,nelgti,nelgvi,nvi 
```

Comment:
Is this a left over debug statement?

Reply:
Necessary to print the header info right away -  critical for large problems while nelgt/nelgv/lelt is printed long after parCon & parRSB. Cosmetic can change if 'hdr calling' lloks like debug stat. Suggestion?